CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 - Added a new API call, `PUT /snapshot/load`, for loading a snapshot.
 - Added new jailer command line argument `--cgroup` which allow the user to
   specify the cgroups that are going to be set by the Jailer.
+- Added full support for AMD CPUs (General Availability).
```

Comment:
I would also add more details or a reference to our readme section where we say what AMD cpus we support

Reply:
done. added a link to the supported platforms